.github/workflows/bump.yml

```diff
@@ -48,21 +47,37 @@ jobs:
           passphrase: ${{ secrets.PHYLUM_BOT_GPG_PASSPHRASE }}
           git_user_signingkey: true
           git_commit_gpgsign: true
-          git_config_global: true
 
-      - name: Set up Homebrew
-        uses: Homebrew/actions/setup-homebrew@master
+      - name: Update URL and SHA256 values
+        id: commit
+        continue-on-error: true
+        # NOTE: The `archive/refs/tags` link is used instead of the `tarball_url` provided by the release API because
+        #       the hashes between the two are (currently) different, even though the extracted contents are the same.
+        #       The `archive/refs/tags` link format is used by the formula and is therefore the one downloaded here.
+        #       Reference: https://github.com/community/community/discussions/45830
+        # NOTE: The git user name/email used for commits is already configured, by crazy-max/ghaction-import-gpg action.
+        run: |
+          curl -sSfLo "cli-${CLI_VER#v}.tar.gz" "https://github.com/phylum-dev/cli/archive/refs/tags/${CLI_VER}.tar.gz"
+          TARBALL_SHA=$(sha256sum "cli-${CLI_VER#v}.tar.gz" | cut -d' ' -f1)
+          sed -E -i '' "/^  url /s/\/v.*.tar.gz/\/${CLI_VER}.tar.gz/" Formula/phylum.rb
+          sed -E -i '' "/^  sha256 /s/.*/  sha256 \"${TARBALL_SHA}\"/" Formula/phylum.rb
+          git add Formula/phylum.rb
+          git commit -m "phylum ${CLI_VER#v}"
+          git push --force origin "HEAD:bump-phylum-${CLI_VER#v}"
+
+      - name: Create Pull Request
+        if: ${{ steps.commit.outcome == 'success' }}
+        uses: actions/github-script@v6
         with:
-          debug: true
           # This PAT is for the `phylum-bot` account and only has the `public_repo` scope to limit privileges.
-          token: ${{ secrets.GH_RELEASE_PAT }}
-
-      - name: Create a PR with new formula URL and SHA
-        env:
-          HOMEBREW_DEVELOPER: "1"
-          HOMEBREW_GITHUB_API_TOKEN: ${{ secrets.GH_RELEASE_PAT }}
-        run: |
-          git remote set-url origin git@github.com:"$GITHUB_REPOSITORY".git
-          CLI_VER_WITHOUT_v=$(echo ${{ github.event.client_payload.CLI_version }} | sed 's/v//')
-          printf "CLI_VER_WITHOUT_v: %s\n" "$CLI_VER_WITHOUT_v"
-          brew bump-formula-pr --version "$CLI_VER_WITHOUT_v" --no-fork --no-browse --no-audit --debug --verbose phylum
+          github-token: ${{ secrets.GH_RELEASE_PAT }}
+          script: |
+            const response = await github.rest.pulls.create({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              head: "bump-phylum-${CLI_VER#v}",
+              base: context.ref,
+              title: "phylum ${CLI_VER#v}",
+              body: "Wait for a new commit to be automatically added by first set of jobs _before_ approving this PR.",
```

Comment:
This sentence doesn't really parse for me. What do you mean here?

Reply:
The text of the PR (the body) was updated to be more verbose, explaining in detail the process for reviewing and merging the PR. The original text was unnecessarily terse due simply to wanting to fit it all on one line (120 character limit).